.travis.yml

```diff
@@ -0,0 +1,68 @@
+conditions: v1
+
+git:
+  depth: false
+
+
+dist: xenial
+cache: pip
+
+
+language: python
+python:
+- "3.7"
+- "3.6"
+- "nightly"  # currently, it's 3.8-dev
+
+
+install:
+- python -m pip install --upgrade flit pip
+- flit install
+
+script:
+- pytest
+
+
+jobs:
+  include:
+  - name: Python 3.7 under Windows
+    os: windows
+    language: sh
+    python: "3.7"
+    before_install:
+    - choco install python --version 3.7
+    env:
+      PATH: >-
+        /c/Python37:/c/Python37/Scripts:$PATH
+
+  - name: Publish dists to production PyPI
+    stage: Publish dists to PYPI
+    if: tag IS present
+    python: "3.7"
+    script:
+    - flit build
+    before_deploy:
+    # Add an empty setup.py stub, because Travis' pypi provider always
+    # calls it and will fail if it's missing. It won't actually get
+    # bundled into dists.
+    - touch setup.py
+    deploy:
+      provider: pypi
+      # `skip-cleanup: true` is required to preserve binary wheel and sdist,
+      # built by during `install` step above.
+      skip-cleanup: true
+      # `skip-existing: true` is required to skip uploading dists, already
+      # present in PyPI instead of failing the whole process.
+      # This happenes when other CI (AppVeyor etc.) has already uploaded
+      # the very same dist (usually sdist).
+      skip-existing: true
+      user: &pypi-user core-workflow
+      password: &pypi-password
+        # Encrypt with `travis encrypt -r python/core-workflow --org` while using travis-ci.org;
```

Comment:
```suggestion
        # Encrypt with `travis encrypt -r python/cherry-picker --org` while using travis-ci.org;
```

Reply:
Actually this part is not needed. We're using travis-ci.com

Reply:
Well, it's good for config readers in the future. For educational purposes, for example :)